os/adding-swap.md

```diff
@@ -0,0 +1,90 @@
+# Adding swap in CoreOS/
+Swap is the process of moving pages of memory to a designated part of the hard disk, freeing up space when needed. Swap can be used to alleviate problems with low-memory environments.
+
+By default CoreOS does not include a partition for swap, however one can configure their system to have swap.
+
+## Manually adding a swapfile on ext4
```

Comment:
You don't need this section; these steps are all performed right in the unit file you added at line 42 below.

You just need to write out the process you tested this afternoon. In short: clean CoreOS box. Drop in `/etc/systemd/system/swap.service` as below, `systemctl enable` and `start` it, and you have swap. Show output of `free` maybe to illustrate.

Then briefly advise on locations and filesystems required.